hist/hist/src/TEfficiency.cxx

```diff
@@ -951,11 +944,10 @@ fPaintGraph(0),
 fPaintHisto(0),
 fWeight(kDefWeight)
 {
-   Bool_t bStatus = TH1::AddDirectoryStatus();
-   TH1::AddDirectory(kFALSE);
+   // do not add cloned histogram to gDirectory
```

Comment:
Minor comment.  Are those really "cloned" histograms or should the comment (here and a couple of other place) be tweaked?

Reply:
Thank you for the comment. Fixed now in the latest commit